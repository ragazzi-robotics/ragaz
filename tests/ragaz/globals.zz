var A, B, C, D, E = True, 'a', 1979, 27.11, "David Ragazzi"
var F: int, G: float = 1.5, 2


def main():
    print(A)
    print(B)
    print(C)
    print(D)
    print(E)
    print(F)
    print(G)
