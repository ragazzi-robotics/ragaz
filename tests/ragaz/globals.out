True
a
1979
27.110000
David Ragazzi
1
2.000000
